assets/sass/4_blocks/_mode-context.scss

```diff
@@ -381,6 +381,10 @@
 
                 .mode-context-title {
                     white-space: normal;
+
+                    a{
+                        font-size: large;
```

Comment:
@Sunairaa So sorry for the late review. I think we should use `word-break: break-word` here instead. Since we don't know the deployment-titles this text-size might not be enough to show the whole word.

Reply:
that seems a good idea to me, I committed changes, can you check my PR again?